mewarpx/tests/test_particle_scraping_stats.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
This test can be removed in favor of the test that will go with the upstream PR. Once we have the FluxDiagnostic for scraped particles implemented we will add a test for that in mewarpx that will test this functionality as well.